apps/gobstopper_service/lib/gobstopper.service/auth/identity.ex

```diff
@@ -61,6 +61,20 @@ defmodule Gobstopper.Service.Auth.Identity do
         end
     end
 
+
+    @doc """
+        Refresh a token of an identity
+
+        Returns `{ :ok, token }` on successful refresh. Otherwise returns an error.
```

Comment:
Should be 2 spaces following the `@doc` segment. As 4 will make it a code block in markdown.